src/db/scripts/fetchTags.js

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable no-console */
+
+// The script fetches Tags from Google Spreadsheet and updates DB accordingly.
+// The script creates new tags and updates existing ones using slug as a primary key.
+// The topic can be updated by moving a Tag from one tab to another.
+// The slug cannot be updated using this script.
+// The script never removes Tags from DB. One should do that manually.
+
+// Usage examples:
+// * npm run fetch-tags
+// * npm run fetch-tags -- secret-dev.json
+
+import Joi from 'joi';
+
+import Tag from 'api/tag/model';
+import Topic from 'api/topic/model';
+
+import { TAG_CONTENT_SCHEMA } from 'constants/topic';
+
+import connectDb from 'db';
+
+const keyBy = require('lodash/keyBy');
+const set = require('lodash/set');
+const isEqual = require('lodash/isEqual');
+const omit = require('lodash/omit');
+const uniq = require('lodash/uniq');
+const difference = require('lodash/difference');
+
+const GoogleSpreadsheet = require('google-spreadsheet');
+
+const doc = new GoogleSpreadsheet('1GxVDSZOWyGvtXa_dwmaRcjzWrXq_kjgXCmZEJVrgb10');
+
+const locales = ['be', 'en', 'ru'];
+
+const tagScopes = {
+  themes: { localized: ['title'], unlocalized: ['slug'] },
+  locations: { localized: ['title'], unlocalized: ['slug', 'image'] },
+  times: { localized: ['title'], unlocalized: ['slug'] },
+  personalities: {
+    localized: ['name', 'dates', 'description'],
+    unlocalized: ['slug', 'color', 'image'],
+  },
+};
+
+const parseSheet = (sheet, scopesLocalized = [], scopesUnlocalized = []) =>
+  new Promise(resolve => {
+    const result = [];
+    sheet.getRows({}, (error, rows) => {
+      rows.shift(); // This removes the "comment" row.
+      rows.forEach(row => {
+        const data = {};
+        scopesUnlocalized.forEach(scope => {
+          const value = row[scope];
+          if (value) {
+            set(data, [scope], row[scope]);
+          }
+        });
+        locales.forEach(locale => {
+          scopesLocalized.forEach(scope => {
+            const value = row[`${scope}.${locale}`];
```

Comment:
you can use lodash `get`

Reply:
it's not the same. I can't use it as `get(row, [scope, locale])` if that's what you mean